WordPress/Classes/ViewRelated/Media/SiteMedia/Controllers/SiteMediaCollectionViewController.swift

```diff
@@ -95,7 +95,7 @@ final class SiteMediaCollectionViewController: UIViewController, NSFetchedResult
     }
 
     private func configureCollectionView() {
-        collectionView.register(SiteMediaCollectionCell.self, forCellWithReuseIdentifier: Constants.cellID)
+        collectionView.register(cell: SiteMediaCollectionCell.self)
```

Comment:
Did you mean to do

```
collectionView.register(SiteMediaCollectionCell.self, forCellWithReuseIdentifier: SiteMediaCollectioinCell.defaultReuseID)
```

Looks like you're using a extension method defined in Kanvas

```
    func register<T: UICollectionViewCell>(cell cellType: T.Type) {
        register(cellType, forCellWithReuseIdentifier: cellType.identifier)
    }
```

This works, because `UICollectionReusableView` conforms to `Kanvas.Identifiable`. Just wanted to point out that it's not leveraging `Reusable`, but `Kanvas.Identifiable` instead

Reply:
Hmm, `Kanvas.Identifiable` is nice, but we should probably not rely on it. I want to get this merged before the `trunk` gets too far ahead. Let me address it separately together with the `filename` fix.